src/System.Private.CoreLib/shared/System/Resources/ResourceManager.cs

```diff
@@ -483,7 +488,7 @@ protected virtual ResourceSet InternalGetResourceSet(CultureInfo culture, bool c
                 // that had resources.
                 foreach (CultureInfo updateCultureInfo in mgr)
                 {
-                    AddResourceSet(localResourceSets, updateCultureInfo.Name, ref rs);
+                    AddResourceSet(localResourceSets, updateCultureInfo.Name, ref rs!); // TODO-NULLABLE: ! should not be required
```

Comment:
https://github.com/dotnet/roslyn/issues/34874

Reply:
do we have any places with quick reference (example pattern and ref to bug)? If not I'm willing to create an issue or update the existing corefx issue so we can look it up quicker

Reply:
We've been including the issue link in the comment.